app/models/manageiq/providers/amazon/instance_types.rb

```diff
@@ -342,6 +342,150 @@ module ManageIQ::Providers::Amazon::InstanceTypes
       :vpc_only                => true,
     },
 
+    "m5.large"    => {
```

Comment:
@rh-dluong - Try moving the => over to the right to align with the hash rockets inside the hash

Reply:
Hm, that didn't seem to work either.  Thanks for the suggestion though!